### PR TITLE
maint: niworkflows 1.4.x compatibility

### PR DESCRIPTION
**Pull request by eilidhmacnicol**
## Changes proposed in this pull request
compatibility for niworkflows 1.4.x series. First half of the changes originally suggested in #35.

### CODE REVIEW

**Comment by eilidhmacnicol** [comment]
STC logic is updated as in commit 820dea4befcca9ea7e2db18e70391937947d8cfd to address edge cases of single volume bolds, due to the removal and refactoring of  `_get_vols_to_discard` from `niworkflows.interfaces.registration`

**Comment by eilidhmacnicol** [comment]
After running this locally I can't reproduce the `3dvolreg` error that caused the smoke test to fail. @oesteban - what do you think?

**Comment by eilidhmacnicol** [comment]
Excellent news - the bug has been caught by the extended artefacts on circleCI. 

The reason I couldn't replicate the issue was because the issue was not in these changes, but in an error in the first brain extraction workflow that was propagated until the generation of an average image later in the workflow. I forgot I had caught (and patched) the error locally, but not pushed it to nirodents. 

The brain extraction workflow's new clip feature (nipreps/nirodents#45) was being followed by denoising, which would introduce voxel values outside of the intended 0-255 range. These values were causing severe issues with the initial INU correction that derailed the brain extraction and consequently the brain reference workflow. Now, with the merge of nipreps/nirodents#49, coregistration is successful between the initial brain reference and the structural, so the later generation of average images are also successful (and not throwing `3dvolreg` errors). 

Long story short, I think this is ready to merge @oesteban

File: setup.cfg [hunk 1]
```diff
@@ -25,16 +25,16 @@ install_requires =
...
+    nirodents >= 0.2.4
     numpy
     pandas
     psutil >= 5.4
     pybids >= 0.11.1
     pyyaml
     requests
     smriprep >= 0.8.0rc0
-    svgutils < 0.3.2
+    svgutils >= 0.3.4
```
Comment (oesteban) [inline setup.cfg:37]:
```suggestion
```

I don't think we call svgutils within fmriprep-rodents, and the pin is done in niworkflows already. Let's simplify

**Comment by eilidhmacnicol** [comment]
Okay - with the nitransforms pin, this is passing all its tests again so I'm going to go ahead and merge.
